modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/radio/RadioDDMFormFieldValueRenderer.java

```diff
@@ -42,8 +43,10 @@ public String render(DDMFormFieldValue ddmFormFieldValue, Locale locale) {
 				ddmFormFieldValue, locale));
 	}
 
-	@Reference
-	protected RadioDDMFormFieldValueAccessor radioDDMFormFieldValueAccessor;
+	@Reference(
+		target = "(component.name=com.liferay.dynamic.data.mapping.form.field.type.internal.radio.RadioDDMFormFieldValueAccessor)"
```

Comment:
@RenanUmV Please double check if this field is still needed, since you've replaced its usage with the util.